docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -135,40 +134,41 @@ image::create-api-designer-design.png[Image of page to create an API design]
 . Select the design title and click the pen icon. Change the design title to *Orders* and click the *Save changes* icon.
 . Click the *Design* tab, and optionally edit the design as follows:
 .. Provide a version number and a description.
-.. AsyncAPI only: Define terms of service.
+.. _AsyncAPI only:_ Define terms of service.
 .. Add your contact information (name, email address, and URL).
 .. Select a license.
-.. OpenAPI only: Define tags.
+.. _OpenAPI only:_ Define tags.
 .. Define one or more servers.
 .. Configure a security scheme.
-.. OpenAPI only: Specify security requirements.
-.. OpenAPI only: Configure vendor extensions.
+.. _OpenAPI only:_ Specify security requirements.
+.. _OpenAPI only:_ Configure vendor extensions.
 +
 [.screencapture]
 .Editing an API design
 image::api-designer-editor.png[Image of API design editor]
 +
 . Click the *Source* tab to see a live preview of the design.
 The content of the *Source* tab automatically updates when you edit values on the editor page.
-
-. Optional: In the left pane of the design editor, you can add the following items:
-.. OpenAPI: Resource paths, data types, and responses
-.. AsyncAPI: Channels, data types, messages, operation traits, and message traits
+. _Optional:_ To see the changes that you made since the last save, click *Actions > Show changes*.
+. _Optional:_ In the left pane of the design editor, you can add the following items:
+.. _OpenAPI only_: Resource paths, data types, and responses
+.. _AsyncAPI only_: Channels, data types, messages, operation traits, and message traits
 . Click *Save*.
 
-The new design is listed on the designs page. You can use the options icon (three vertical dots) to open the design editor, download the design to a file, or delete the design.
+The new design is listed on the *API and Schema Designs* page.
```

Comment:
I think passive voice is acceptable in this case because the system is performing the action.
A "good" example from IBM Style is similar: "The file is saved when you press Enter."

Reply:
Yes, sorry, I tend to suggest active voice wherever I see a reasonable way of writing it. But I recognize that passive voice is also acceptable here.